pdfminer/high_level.py

```diff
@@ -88,3 +90,46 @@ def extract_text_to_fp(inf, outfp,
         interpreter.process_page(page)    
 
     device.close()
+
+
+def pdf_to_text(pdf_file, password="", page_numbers=set(), maxpages=0,
+                caching=True, codec="utf-8"):
```

Comment:
I know I am a bit capricious on the name of this function, but currently I think that your suggestion to name it `extract_text` is better. 

I was thinking about the other high-level functions we are going to add here and think that the extract_* name pattern will be more consistent: 

- Text as html: `extract_text_as_html()` (or `pdf_to_html()`?)
- Table of contents: `extract_toc()`. Using `pdf_to_toc()` is just weird.
- Images: `extract_images()`. Using `pdf_to_images()` is also weird.

Reply:
I also did prefer my suggestions, but opted to go with yours haha
Fixed that